### PR TITLE
added Find Place API call

### PR DESCRIPTION
**Pull request by rossiter10**
(2nd attempt) 
Added the "Find Place" API call, request, and response classes.
https://developers.google.com/places/web-service/search#FindPlaceRequests

And added one unit test for now so something was there at least.

### CODE REVIEW

File: GoogleMapsApi.Test/IntegrationTests/PlacesFindTests.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                ApiKey = ApiKey,
+                Input = "hilton chicago",
+                InputType = InputType.TextQuery
+            };
+
+            PlacesFindResponse result = GoogleMaps.PlacesFind.Query(request);
+
+            AssertInconclusive.NotExceedQuota(result);
+            Assert.AreEqual(Status.OK, result.Status);
+            Assert.AreEqual("ChIJL3osJJksDogRodsJu9TjTQA", result.Candidates.FirstOrDefault()?.PlaceId);
```
Comment (maximn) [inline GoogleMapsApi.Test/IntegrationTests/PlacesFindTests.cs]: This is very fragile, in a while the first candidate will have another ID and this test will break.

You can do something like 
- have more than 1 candidate
- the first candidate has an ID longer than x (10?) characters

**Comment by maximn** [comment]
Nuget 0.78

**Comment by maximn** [comment]
Did you run these tests? Looks like failing - https://travis-ci.org/maximn/google-maps/builds/466175042

**Comment by rossiter10** [comment]
Crap, sorry. I have them passing now in my branch. Do I need to do a separate PR?

**Comment by maximn** [comment]
Nuget - 0.78.0

**Pull request merged**